File: Fundamentals/Data Types and Variables - Exercise/triples_of_latin_letters.py
end_range = int(input())
for first_char in range(97, 97 + end_range):
    for second_char in range(97, 97 + end_range):
        for third_char in range(97, 97 + end_range):
            print(f'{chr(first_char)}{chr(second_char)}{chr(third_char)}')

File: JS Fundamentals/Data Types and Variables - Lab/triples_of_latin_letters.js
function looping(numb){
    numb = Number(numb);
    for(let first_char = 97; first_char < 97 + numb; first_char++){
        for(let second_char = 97; second_char < 97 + numb; second_char++){
            for(let third_char = 97; third_char < 97 + numb; third_char++){
                console.log(`${String.fromCharCode(first_char)}${String.fromCharCode(second_char)}${String.fromCharCode(third_char)}`);
            }
        }
    }
}

looping('3');
looping(2);